lib/widget/input/form_textfield.dart

```diff
@@ -317,6 +316,14 @@ class TextInputState extends FormFieldWidgetState<BaseTextInput>
         }
       }
     });
+
+    // Checking for readOnly from parent widget and assign the value to TextInput and PasswordInput if it's readOnly property is null
+    final formController =
+        context.findAncestorWidgetOfExactType<EnsembleForm>()?.controller;
+
+    if (formController != null && widget._controller.readOnly == null) {
+      widget._controller.readOnly = formController.readOnly == true;
```

Comment:
You are inheriting this property from Form, so it's not proper to overwrite it. This cause unnecessary bugs if the user is reading it back, and also waste another rendering cycle. You should simple do a getter `isReadOnly() => isReadOnly ?? form.isReadOnly`

Reply:
How can I get EnsembleForm widget's `formController` to get the readOnly property?

Our TextInput and PasswordInput field's controller extends from (`TextInputController -> FormFieldController -> WidgetController`)
But the Form widget's controller extends from (`FormController -> WidgetController`).

Now I want to get the `Form widget's readOnly` property and to set that parent value (Form's readOnly) to the children's (TextInput or PasswordInput) field's `readOnly` property